static-assets/components/cstudio-forms/template-editor.js

```diff
@@ -94,37 +94,42 @@ CStudioAuthoring.Module.requireModule(
 
               Promise.all([
                 new Promise((resolve) => {
-                  CStudioAuthoring.Service.getConfiguration(
+                  CrafterCMSNext.services.configuration.fetchFileDOM(
                     CStudioAuthoringContext.site,
                     "/code-editor-config.xml",
-                    { success: resolve }
+                    "studio"
+                  ).subscribe(
```

Comment:
call `.toPromise` from the service return directly instead of wrapping in a promise, subscribing and sending resolve.